numpy/lib/arraypad.py

```diff
@@ -986,399 +136,191 @@ def _validate_lengths(narray, number_elements):
     return normshp
 
 
-###############################################################################
-# Public functions
-
-
-def pad(array, pad_width, mode, **kwargs):
+def _arange_ndarray(shape, axis, reverse=False):
     """
```

Comment:
Any reason you cut this docstring? Leaving it in would result in a more easy to read diff

Reply:
Ah, sorry, the way GitHub displayed the diff confused me. I see your point now. I'll have a look if I can salvage something of `_arange_ndarray`s old docstring.

Reply:
I don't think there is much so salvage as this isn't really the same function anymore. While the old function behaved more like `np.arange` the new one includes arguments like `start` and `stop` and is closer to `linspace`...